weedle2/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "weedle2"
-version = "3.0.0"
+version = "4.0.0"
```

Comment:
If we don't change the public API nor export any nom types we can make this a minor release.